README.md

```diff
@@ -78,6 +78,27 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
```

Comment:
This readme change is out of scope of this PR, as well as should be handled differently.

Rather, docs should have a subsection specifically on `ficsit-cli` just like it does for SMM: https://docs.ficsit.app/satisfactory-modding/latest/ForUsers/SatisfactoryModManager.html and the readme should reference that instead.

Please remove.

Reply:
Unless and until the docs have that relevant section, this README is the **only** place to discover that

- ficsit-cli, unlike SMM, does not auto-detect installations
- manually specifying the installation is a _necessary_ pre-requisite to using the app
- What the correct value for that will be in 99% of cases

Given that there's no way to add an installation via the interactive menus, it actually too me longer to figure out how to make the app work than it did to make the change.

This is the "boy scout" principle, leave a codebase in the state you would like to have found it, to make life easier for those that follow. I'm not at all comfortable reverting a change that achieves that, especially when the information it contains isn't something that's realistically ever going to change.

Reply:
I'm not sure where you got the idea that it's not possible to add an installation via the interactive menu? I added a video below of how to do it.

As for the documentation, then please either go the extra length and include basic default instructions for all systems, or none. The most likely audience for this specific tool are Linux users running it on a headless server, where an SMM UI is impossible to be rendered.

I agree that the readme should be more informational, as it's currently even outdated for the paths it does provide, as it's missing the Linux equivalents, but also, this project is not in a released condition, but rather a 0-semver version, which should be considered unstable and not ready for general public use.

In general your message comes across quite passive-aggressive and condescending, which I am really not a fan of.

In either case, in its current state, I am not approving this PR.

(ignore weird colors for logo, I'm messing around with my terminal)

https://github.com/satisfactorymodding/ficsit-cli/assets/1759390/3fa176be-721b-4dc5-af9a-59b5531bcafd

Reply:
Honestly, it's mostly frustration.

This PR is designed specifically for MacOS/Crossover users. I had to actively track down
- the fact that I had to manually add an installation
- the default installation path for satisfactory
- whether or not I needed to use the base `Satisfactory` folder or the nested `FactoryGame` folder

Earned knowledge that I chose to pay forward to future people who wanted to use this app. Requests for changes that essentially say "nope, don't tell others that, pass on the burden" don't make me feel at all comfortable.